ramalama/version.py

```diff
@@ -5,10 +5,14 @@
 
 def version():
     try:
-        return importlib.metadata.version("ramalama")
+        ver = importlib.metadata.version("ramalama")
```

Comment:
I have to head offline @rhatdan but I'm having diffculties with this version function, if you have an idea what would help, let me know, otherwise I'll poke at it again tomorrow.